src/Docker.jl

```diff
@@ -2,7 +2,8 @@ using Random
 
 struct DockerExecutor <: SandboxExecutor
     label::String
-    DockerExecutor() = new(Random.randstring(10))
+    privileges::Union{Symbol, Nothing}
```

Comment:
Why `Symbol` and `Nothing`?  If `nothing` is the default and is the same as `:privileged`, then just make the default value `:privileged` and don't allow `nothing`, I think.

Reply:
Done! I just pushed a commit with that change.

In that same commit, I also added a kwarg constructor, because then it will be easier for users to use this feature, e.g.:
```julia
exe = DockerExecutor(; privileges = :unprivileged)
```